src/sherpa_ai/status_loggers/status_loggers.py

```diff
@@ -0,0 +1,29 @@
+
+from sherpa_ai.status_loggers.base import BaseStatusLogger
```

Comment:
Can you just integrate this logger with the verbose logger? I think add another module for the logger can be a bit confusion.

Reply:
Other parts look good to me